x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/coverage_overview/coverage_overview.cy.ts

```diff
@@ -0,0 +1,429 @@
+
+        openTechniquePanelByName(EnabledCustomRuleMitreData.technique.name);
+        cy.get(COVERAGE_OVERVIEW_POPOVER_ENABLED_RULES).contains('Enabled custom rule');
+
+        openTechniquePanelByName(DisabledCustomRuleMitreData.technique.name);
+        cy.get(COVERAGE_OVERVIEW_POPOVER_DISABLED_RULES).contains('Disabled custom rule');
+      });
+
+      it('filters for search term', () => {
+        filterCoverageOverviewBySearchBar('Enabled custom rule'); // Disables default filter
+
+        openTechniquePanelByName(EnabledPrebuiltRuleMitreData.technique.name);
+        cy.get(COVERAGE_OVERVIEW_POPOVER_ENABLED_RULES)
+          .contains('Enabled prebuilt rule')
+          .should('not.exist');
+
+        openTechniquePanelByName(DisabledPrebuiltRuleMitreData.technique.name);
+        cy.get(COVERAGE_OVERVIEW_POPOVER_DISABLED_RULES)
+          .contains('Disabled prebuilt rule')
+          .should('not.exist');
+
+        openTechniquePanelByName(EnabledCustomRuleMitreData.technique.name);
+        cy.get(COVERAGE_OVERVIEW_POPOVER_ENABLED_RULES).contains('Enabled custom rule');
+
+        openTechniquePanelByName(DisabledCustomRuleMitreData.technique.name);
+        cy.get(COVERAGE_OVERVIEW_POPOVER_DISABLED_RULES)
+          .contains('Disabled custom rule')
+          .should('not.exist');
+      });
+    });
+
+    it('enables all disabled rules', () => {
+      selectCoverageOverviewActivityFilterOption('Disabled rules');
+      openTechniquePanelByName(DisabledPrebuiltRuleMitreData.technique.name);
+      enableAllDisabledRules();
+
+      // Should now render all rules in "enabled" section
+      openTechniquePanelByName(DisabledPrebuiltRuleMitreData.technique.name);
+      cy.get(COVERAGE_OVERVIEW_POPOVER_ENABLED_RULES).contains('Disabled prebuilt rule');
+
+      // Shouldn't render the rules in "disabled" section
+      cy.get(COVERAGE_OVERVIEW_POPOVER_DISABLED_RULES)
+        .contains('Disabled prebuilt rule')
+        .should('not.exist');
+      cy.get(COVERAGE_OVERVIEW_ENABLE_ALL_DISABLED_BUTTON).should('be.disabled');
+    });
+  });
+
+  describe('with rules that have identical mitre techniques that belong to multiple tactics', () => {
+    beforeEach(() => {
+      login();
+      deleteAlertsAndRules();
+      deletePrebuiltRulesAssets();
+      createRule(
+        getNewRule({
+          rule_id: 'duplicate_technique_rule_1',
+          enabled: true,
+          name: 'Rule with tactic 1',
+          threat: [MockCustomRuleDuplicateTechniqueThreat1],
+        })
+      );
+      createRule(
+        getNewRule({
+          rule_id: 'duplicate_technique_rule_2',
+          name: 'Rule with tactic 2',
+          enabled: false,
+          threat: [MockCustomRuleDuplicateTechniqueThreat2],
+        })
+      );
+      visit(RULES_COVERAGE_OVERVIEW_URL);
+    });
+
+    it('technique panels render unique rule data', () => {
+      selectCoverageOverviewActivityFilterOption('Disabled rules');
```

Comment:
This could be super misleading for anyone who doesn't remember what filters are enabled on the page by default. Let's add a comment that by selecting "Disabled rules" we actually make both enabled and disabled rules shown on the page.